src/einsteinpy/plotting/geodesics/static.py

```diff
@@ -60,6 +73,16 @@ def _draw_attractor(self, radius, xarr, yarr):
                 Circle((0, 0), radius.value, lw=0, color=self.attractor_color)
             )
 
+    def _set_scaling(self, x_range, y_range, z_range, lim):
+        if x_range < lim and y_range < lim and z_range < lim:
```

Comment:
I added this so that if all ranges are less than the limit then we should not change the scale of axes as everything would then become too small.

Reply:
RIght